hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/lease/LeaseManager.java

```diff
@@ -235,7 +235,7 @@ public void run() {
           }
         } catch (InterruptedException e) {
           // This means a new lease is added to activeLeases.
-          LOG.error("Execution was interrupted ", e);
+          LOG.debug("Execution was interrupted ", e);
```

Comment:
This should not be related to block deletion. What exception are we seeing here?